lib/modules/manager/pip-compile/artifacts.spec.ts

```diff
@@ -28,6 +28,18 @@ jest.mock('../../../util/host-rules', () => mockDeep());
 jest.mock('../../../util/http');
 jest.mock('../../datasource', () => mockDeep());
 
+const requirementsWithUv = `# This file was autogenerated by uv via the following command:
```

Comment:
please move to test if only used once next time and use `codeBlock` template helper